src/Nest/Nest.csproj

```diff
@@ -12,4 +12,7 @@
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="XPack\Ssl" />
+  </ItemGroup>
```

Comment:
Superfluous, I think

Reply:
yeah, annoying not sure what IDE action causes these to pop up yet.